style.css

```diff
@@ -18,90 +20,131 @@ body {
   }
 }
 
+a {
+  color: black;
+  text-decoration: none;
+}
+
 .navbar {
```

Comment:
- [ ] Kindly make sure you stick to the design as much as possible.
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/201911597-90a57db4-ee37-4eec-928b-95f19def6c39.png)
Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/201911686-84765b4f-7e89-424b-93f6-f0a60d8b0f35.png)